cats/cats.py

```diff
@@ -271,9 +285,12 @@ def cli(
     else:
         final_bundle_dump = json.dumps(final_bundle.to_json_dict(), sort_keys=True, indent=4)
 
-    confirmation = input(
-        "The transaction has been created, would you like to push it to the network? (Y/N)"
-    ) in ["y", "Y", "yes", "Yes"]
+    confirmation = pushtx
+
+    if not quiet:
+        confirmation = input(
+            "The transaction has been created, would you like to push it to the network? (Y/N)"
+        ) in ["y", "Y", "yes", "Yes"]
     if confirmation:
         response = asyncio.get_event_loop().run_until_complete(
             push_tx(fingerprint, final_bundle)
```

Comment:
It looks like push_tx may interfere with this function name here.

Reply:
It didn't interfere in my tests but I renamed it just in case